main.tf

```diff
@@ -1,13 +1,96 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  s3_bucket_job_source_name = module.s3_bucket_job_source.bucket_id
+  role_arn                  = module.iam_role.arn
+}
+
+module "s3_bucket_job_source" {
+  source  = "cloudposse/s3-bucket/aws"
+  version = "2.0.3"
+
+  acl                          = "private"
+  versioning_enabled           = false
+  force_destroy                = true
+  allow_encrypted_uploads_only = true
+  allow_ssl_requests_only      = true
+  block_public_acls            = true
+  block_public_policy          = true
+  ignore_public_acls           = true
+  restrict_public_buckets      = true
+
+  context = module.this.context
+}
+
+module "iam_role" {
+  source  = "cloudposse/iam-role/aws"
+  version = "0.16.2"
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+  principals = {
+    "Service" = ["glue.amazonaws.com"]
   }
+
+  managed_policy_arns = [
+    "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
+  ]
+
+  policy_document_count = 0
+  policy_description    = "Policy for AWS Glue with access to EC2, S3, and Cloudwatch Logs"
+  role_description      = "Role for AWS Glue with access to EC2, S3, and Cloudwatch Logs"
```

Comment:
Let's expose these inputs with these strings as the defaults. It's often the case that sometimes one wants to modify them

Reply:
We spoke a bit about this in slack.

> ok i think i understand how to handle this. if you remove all the *.tf files in the root (except versions.tf), then bats will be OK with it
> if not, then let’s add a comment stating in the README and in each root tf file that this is purely an example and a temporary bats workaround